flink-core/src/main/java/org/apache/flink/api/common/accumulators/Accumulator.java

```diff
@@ -1,20 +1,4 @@
-/*
```

Comment:
Accidentally removal I think. Revert to pass the checker :-)

Reply:
Thanks. I will revert this before merge. This is used to fail the travis fast.

Reply:
Aha, I think we will back to the thread reduce building time :-)